redhat/rdma-core.spec

```diff
@@ -35,6 +35,13 @@ BuildRequires: python3
 BuildRequires: python
 %endif
 %endif
+
+%if 0%{?rhel} >= 8 || 0%{?fedora} >= 30 || %{with_pyverbs}
+BuildRequires: python3-docutils
+%else
+BuildRequires: python-docutils
+%endif
```

Comment:
Lets just put this inside the existing pandoc stanza and update the FC version to 30.

Not sure what the pyverbs is all about either??

Reply:
> Not sure what the pyverbs is all about either??
Are you talking about "|| %{with_pyverbs}"?
I added it, because with_pyverbs we will install python3 without any relation to fedora version.

Reply:
> Lets just put this inside the existing pandoc stanza and update the FC version to 30.
> 

Because I need with_pyverbs check, putting inside pandoc looks extremely ugly.
Pandoc doesn't need with_pyverbs, while python3-docutils needs.